src/Application/Console/Formatters/FormatResolver.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Console\Formatters;
+
+use InvalidArgumentException;
+use NunoMaduro\PhpInsights\Application\Console\Contracts\Formatter;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @internal
+ */
+final class FormatResolver
+{
+    /**
+     * @var array<string>
```

Comment:
I don't really know this notation, but in this case, that should be `array<string, string>` ?

Reply:
Totally true. In the start the array only had the classes, forgot to update the annotation, thanks!

Reply:
Don't forget to update this :wink: